utils/build-script

```diff
@@ -700,6 +705,41 @@ the number of parallel build jobs to use""",
     a '\'"""),
         action="append", dest="extra_swift_args", default=[])
 
+    android_group = parser.add_argument_group(
+        title="Build settings for Android")
+    android_group.add_argument(
+        "--android-ndk",
+        help="An absolute path to the NDK that will be used as a libc "
+             "implementation for Android builds",
```

Comment:
I'll have to try out the latest NDK, thanks for the tip! If the paths have changed then I'll make sure we support both.

Not sure about paradigms, but if you're asking whether this works, I would say it does! You should be able to run the test suite yourself by following the instructions in #1714.